.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## Description
+
+Please describe the changes in your pull request in few words here.
+
+### Changes
+
+List the changes done to fix a bug or introducing a new feature.
+
+## How to test
+
+Describe the steps required to test the changes proposed in the pull request.
+
+Please consider using the closing keyword if the pull request is proposed to
+fix an issue already created in the repository
+(https://help.github.com/articles/closing-issues-using-keywords/)
```

Comment:
Please change the default template for documents and make this template for non-document PRs.

The document PR can be like

```md
## Project

Name of the project

### Changes

What was updated (new pages, weekly update, etc.)

### Notes

If there are any further notes.
```

Reply:
Okay @GMishx updated